src/contract_wrappers/exchange_wrapper.ts

```diff
@@ -150,13 +150,18 @@ export class ExchangeWrapper extends ContractWrapper {
         assert.doesConformToSchema('signedOrder', signedOrder, schemas.signedOrderSchema);
         assert.isBigNumber('fillTakerTokenAmount', fillTakerTokenAmount);
         assert.isBoolean('shouldThrowOnInsufficientBalanceOrAllowance', shouldThrowOnInsufficientBalanceOrAllowance);
+        // this is already being done in validateFillOrderThrowIfInvalidAsync
```

Comment:
We'd still want to do it here too in case this method gets refactored and no longer calls `validateFillOrderThrowIfInvalidAsync `

Reply:
hmm, lemme think on this, might be a way to save some rpcs